src/plugins/geomap/geomap-en.hbs

```diff
@@ -1303,3 +1303,53 @@ var wet_boew_geomap = {
 		</details>
 	</section>
 </section>
+
+<section id="aoiZoomOnLoad">
+	<h2>Zoom to AOI on page load</h2>
```

Comment:
Could you add the word extent in parathensis here since you are interchangeably using zoom AOI and extent for this feature?
```suggestion
	<h2>Zoom to AOI on page load (extent)</h2>
```